GPU/Common/SoftwareTransformCommon.cpp

```diff
@@ -354,6 +351,8 @@ void SoftwareTransform(
 
 			case GE_TEXMAP_TEXTURE_MATRIX:
 				{
+					// TODO: What's the correct behavior with flat shading?  Provoked normal or real normal?
```

Comment:
My guess would be real normal, flat shading is not really related to the texture pipe at all I think..